lib/slack/util.py

```diff
@@ -3,9 +3,11 @@
 import igraph
 import lib.slack.config as config
 import pickle
+
```

Comment:
The global variable needs to be removed.

Reply:
was unused. removed it